app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="SquareImageDimension">64dp</dimen>
```

Comment:
This is not the way camelCase is written.. Here it should be `squareImageDimension`

Reply:
Fixed